CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * [#215](https://github.com/ruby-grape/grape-entity/pull/217): Fix: `#delegate_attribute` no longer delegates to methods included with `Kernel` - [@maltoe](https://github.com/maltoe).
 * [#219](https://github.com/ruby-grape/grape-entity/pull/219): Fix: double pass options in serializable_hash - [@sbatykov](https://github.com/sbatykov).
 * [#226](https://github.com/ruby-grape/grape-entity/pull/226): Add fetch method to fetch from opts_hash - [@alanjcfs](https://github.com/alanjcfs).
+* [#231](https://github.com/ruby-grape/grape-entity/pull/231): Fix for #215 allow delegate_attribute for derived enyity - [@sbatykov](https://github.com/sbatykov)
```

Comment:
If you're going to mention #215, link it the same way as the PR, but generally we just don't.

Entity is misspelled.

There needs to be a period at the end of this, see other lines;)